### PR TITLE
aim styling refactor TM-3473

### PR DESCRIPTION
**Pull request by SidelineCory24**
To-do:
- [x] AIM Page should resize seamlessly

**Note: This ticket/pr is only to address the screen resizing issue. There are other tickets to handle the styling for the input fields, button alignment, and showing the research pane/double screen.**

### CODE REVIEW

**Comment by SidelineCory24** [comment]
Before changes
https://user-images.githubusercontent.com/55964163/192377700-84f60e3e-ca1f-476c-9230-3d9708429d63.mp4

After changes

Uploading resizing-aim5.mp4…

**Comment by SidelineCory24** [comment]
After changes
Uploading resizing-aim5.mp4…

File: src/sass/_agendaItemMaintenance.scss [hunk 1]
```diff
@@ -417,38 +413,19 @@
...
-    width: 90%;
-  }
-}
-
-@media screen and (min-width: 1100px) and (max-width: 1200px) {
-  .stacked {
-    width: 77%;
+    @media screen and (max-width: 1600px) {
+      margin-left: 0px;
+    }
```
Comment (elizabeth-jimenez) [inline src/sass/_agendaItemMaintenance.scss:428]: Nice catch!

File: src/sass/_agendaItemMaintenance.scss [hunk 2]
```diff
@@ -252,10 +250,9 @@
   }
 
   .maintenance-container-left-expanded {
-    float: left;
     height: 98vh;
     overflow-y: auto;
-    padding: 0 10px 20px 0;
+    padding-bottom: 20px;
```
Comment (elizabeth-jimenez) [inline src/sass/_agendaItemMaintenance.scss:255]: we can remove this - might look a bit better with no padding. See how the scroll seems to dangle lol
image

**Review by elizabeth-jimenez — CHANGES_REQUESTED** [review]
Looks great! Just a few changes

File: src/sass/_agendaItemMaintenance.scss (hunk 2 above)
Comment (SidelineCory24) [reply src/sass/_agendaItemMaintenance.scss:255]: done

File: src/sass/_agendaItemMaintenance.scss (hunk 1 above)
Comment (SidelineCory24) [reply src/sass/_agendaItemMaintenance.scss:428]: yeah it does and keeps it consistent with the stacked view.

**Comment by SidelineCory24** [comment]
> Looks great! Just a few changes

feedback has been addressed

File: src/sass/_agendaItemMaintenance.scss [hunk 3]
```diff
@@ -274,7 +273,6 @@
   .maintenance-container-right {
     background-color: $color-white;
     height: 98vh;
-    margin-left: 20px;
```
Comment (elizabeth-jimenez) [inline src/sass/_agendaItemMaintenance.scss:281]: You can remove it from below now too, cause 0 and nothing seem to be the same

**Review by elizabeth-jimenez — APPROVED** [review]
🐈

File: src/sass/_agendaItemMaintenance.scss (hunk 3 above)
Comment (SidelineCory24) [reply src/sass/_agendaItemMaintenance.scss:281]: done

**Pull request merged**